.github/workflows/main.yml

```diff
@@ -302,6 +302,18 @@ jobs:
     - name: Prepare output directory
       run: mkdir output_uhdm_vanilla_verilator
 
+    - name: Build & Test (uhdm verilator)
+      run: |
+        pip install virtualenv
+        ./uhdm-integration/.github/ci.sh
+        set -o pipefail
+        ./uhdm-integration/build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator --meminit=rom,uhdm-tests/opentitan/boot_rom_sim_verilator.elf --meminit=flash,uhdm-tests/opentitan/hello_world_sim_verilator.elf | tee output_uhdm_vanilla_verilator/uhdm_verilator_simulation.log
```

Comment:
The name of the binary has to be changed. There is an error that there is no such file or directory.

Reply:
But then it will hang in infinite loop. What do you think about adding a timeout? It will also prevent from hanging in the future.

Reply:
Good catch! Changed and added.